src/components/Link/index.tsx

```diff
@@ -114,7 +136,9 @@ const Link: React.FC<ILinkProps> = ({ href, scrollOptions, ...restProps }) => {
     }
   }
 
-  return <ResultLinkComponent href={href} {...restProps} onClick={onClick} />
+  return mounted ? (
+    <ResultLinkComponent href={href} {...restProps} onClick={onClick} />
+  ) : null
```

Comment:
I'm not so sure about this, does it prevent links from being seen without JS- e.g. from the `lychee` checks we have planned? Do the issues still persist without this state and conditional?

Reply:
You're right! `lychee` checks would break and would stop links from being seen without JS! I updated the code, replacing `null` with the jsx. Essential just telling the component to re render on mount. There's probably a cleaner way to re render a component though 🤔